log/development.log

```diff
@@ -0,0 +1,93 @@
+Started GET "/" for 127.0.0.1 at 2018-11-23 14:59:32 -0300
```

Comment:
You should ignore this files. :thinking:

Reply:
Done! ad0a182